tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/ServerlessMiniAgentManager.cs

```diff
@@ -0,0 +1,91 @@
+// <copyright file="ServerlessMiniAgentManager.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.ServerlessInstrumentation;
+
+internal class ServerlessMiniAgentManager
```

Comment:
It may not make any sense (I'm only doing a quick review and it's late here ;)) but I assume this class is more or less doing the same thing than this one: https://github.com/DataDog/dd-trace-dotnet/blob/fdd89ed264bc8aeb33d9a9c6c14fbb1ada1701d0/tracer/src/Datadog.Trace/AgentProcessManager.cs#L22 (this is the class responsible for handling the trace agent for AAS). Maybe there are opportunities to share things. Maybe not. But wanted to raise it to your attention

Reply:
I've refactored the code so there isn't a "manager" class anymore, it wasn't doing much managing (aside from a single spawn method) so I combined it with the rest.